odoo/addons/base/data/res_partner_demo.xml

```diff
@@ -365,13 +365,15 @@
           <field name="parent_id" eval="ref('main_partner')"/>
           <field name="function">Chief Executive Officer (CEO)</field>
           <field name="name">Mark Davis</field>
+          <field name="company_id" ref="main_company"/>
```

Comment:
could you explain that part?

Reply:
The demo data of partners Administrator, Demo User and Demo Portal User have this line. And I thought that the partners Mark Davis and Roger Scott, that are from the same company, should also have that line. I don't know if this two added lines are what made the PR green or if it was because of another of the changes. I'm as surprised as you because the differences from this PR and the old PR are just stylistic excluding this two lines. 😄

Reply:
Well, the old one has <code>< field name="company_id"></ field></code> in several partners in <code>res_partner_demo.xml</code> and this PR doesn't have it. This PR was taken from your modified commit 😅

Reply:
@mart-e Then you can merge this. And if I find a way to put the empty company in the partners of <code>res_partner_demo.xml</code> without failing any test, then I will do another PR.